### PR TITLE
Update changelog for #1582

### PR DESCRIPTION
**Pull request by Judahmeek**
Updating changelog for #1582 so we can make a new minor release for the type fixes.

- - -
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/1584)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,8 +12,9 @@ We specialize in helping companies quickly and efficiently update client-side de
 Please follow the recommendations outlined at [keepachangelog.com](http://keepachangelog.com/). Please use the existing headings and styling as a guide, and add a link for the version diff at the bottom of the file. Also, please update the `Unreleased` link to compare to the latest release version.
 
 ## Versions
-### [Unreleased]
-Changes since the last non-beta release.
+### [13.4.1]
+#### Fixed
+- Fixed Typescript types for ServerRenderResult, ReactComponent, and RenderFunction interfaces. [PR 1582](https://github.com/shakacode/react_on_rails/pull/1582) by [kotarella1110](https://github.com/kotarella1110)
```
Comment (ahangarha) [inline CHANGELOG.md]: I think we should keep those two lines for the unreleased version intact.

**Review by ahangarha — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Judahmeek) [reply CHANGELOG.md]: Good call. Will fix.

**Review by ahangarha — CHANGES_REQUESTED** [review]
We also need to update version links at the end of the changelog file.

**Comment by Judahmeek** [comment]
> We also need to update version links at the end of the changelog file.

Another good catch. Done.

**Review by ahangarha — APPROVED** [review]

**Pull request merged**